config.js

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {util} = require('bedrock');
+
+const c = util.config.main;
+const cc = c.computer();
+
+cc('paypal', {
```

Comment:
Why are we not naming this configuration `bedrock-payment-paypal`? The convention for config namespace names have usually been their package name. Why are we using a computed config here? I'd assume we use computed configs when these values are dynamic and may change. Please correct me if I'm wrong @davidlehn.

Reply:
Yes, we usually use package name.  Though sometimes omit the 'bedrock-' prefix.

Doesn't look like computed config is needed here since it's just initializing fields.

Reply:
Yeah we went with `payment-http` elsewhere. This is now `bedrock-payment-paypal` https://github.com/digitalbazaar/bedrock-payment-paypal/pull/2/commits/42a81daf80556f5ef82c3744e34cb7533e466d79

anyways will try to use the package name in the future.

Reply:
Could also remove the computed config part.  But no big deal either way.

Reply:
I would prefer to stick with just using computed for all settings. I seem to have issues figuring out when to use computed and when to not so I'm just going to use `cc` whenever I can.